Packs/CommonScripts/Scripts/Ping/Ping_test.py

```diff
@@ -34,3 +34,21 @@ def test_fail_ping(mocker):
     # call_args last call with a tuple of args list and kwargs
     err_msg = return_error_mock.call_args[0][0]
     assert 'Name does not resolve' in err_msg or 'Name or service not known' in err_msg
+
+
+def test_fail_ping_permission_error_xsoar8(mocker):
```

Comment:
luckily is_xsoar_on_prem() returns false :) but it's ok

Reply:
true, i will mock it anyway